src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ReaderMetricsProcessorTests.java

```diff
@@ -44,39 +44,52 @@ public ReaderMetricsProcessorTests() throws SQLException, ClassNotFoundException
     super();
   }
 
-  // Disabled on purpose
-  // @Test
-  public void testReaderMetricsProcessor() throws Exception {
-    String rootLocation = "build/private/test_resources/dev/shm";
+  @Test
+  public void testReaderMetricsProcessorFrequently() throws Exception {
+    String rootLocation = "build/resources/test/reader/";
```

Comment:
nit: Maybe make this an instance variable, or initialize this in before() so that you don't have to do this on every test case.

Reply:
I have updated that in the latest version.